src/main/java/racing/domain/RacingCars.java

```diff
@@ -43,4 +44,11 @@ public int getFarthestPosition() {
                 .max()
                 .orElse(0);
     }
+
+    public List<Car> getWinnerCars() {
+        int farthestPosition = getFarthestPosition();
+        return this.cars.stream()
+                .filter(car -> farthestPosition == car.getPosition())
```

Comment:
Racing#isNotFinished 처럼 자동차에게 위치가 같은지 확인해달라고 해도 좋을 것 같아요 😃 
```suggestion
                .filter(car -> car.samePosition(farthestPosition))
```